decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -138,44 +138,58 @@
   </div>
 </div>
 
-<% if proposal.valuators.any? %>
 <div class="card">
   <div class="card-divider">
     <h2 class="card-title">
       <%= t ".valuators" %>
     </h2>
   </div>
-
   <div class="card-section">
-    <div class="row column">
-      <strong><%= t ".assigned_valuators" %>:</strong>
-      <ul id="valuators">
-        <% proposal.valuation_assignments.each do |assignment| %>
-          <% presented_valuator = present(assignment.valuator) %>
+    <div id="js-assign-proposal-to-valuator-actions" class="js-bulk-action-form form-single-proposal">
```

Comment:
```suggestion
    <% if allowed_to? :assign_to_valuator, :proposals %>
      <div id="js-assign-proposal-to-valuator-actions" class="js-bulk-action-form form-single-proposal">
```

We should only show this menu in the case that the user has permission for it. The helper `allowed_to?` is the equivalent for the views to the `enforce_permission_to` used in controllers.